java-checks-test-sources/src/main/files/non-compiling/checks/SunPackagesUsedCheckCustom.java

```diff
@@ -2,7 +2,7 @@
 import com.sun.jersey.api.client.ClientHandlerException; // Compliant
 import java.util.ArrayList;
```

Comment:
We should add a non compliant import from `com.sun.faces.*` or `com.sun.jersey.*" element to show that the `exclude` pattern is overwritten and not extended on top of its default value.

Reply:
Good idea